index.html

```diff
@@ -1,403 +1,432 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="style.css" rel="stylesheet">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap"
+        rel="stylesheet">
     <title>Conor Follette</title>
 </head>
 
 <body>
+
     <div class="container">
         <header class="mobile_header">
             <div class="logo">
                 <a href="">
-                <img class="logo_picture" src="img/LogoPlaceholder.png" alt="My_Logo">
+                    <img class="logo_picture" src="img/LogoPlaceholder.png" alt="My_Logo">
                 </a>
             </div>
-            <nav class="menu_icon">
+
+
+            <nav class="menu_icon" id="menu_icon">
                 <img id="hamburger_menu" src="img/Union.png" alt="menu_button">
             </nav>
+
+            <aside class="popup_window" id="popup_window">
```

Comment:
- [ ] Kindly check your styling, your background is affecting your menu items check the comparison below:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/179551677-4c1f73eb-6d13-4c07-84b7-63a5597b2066.png) |  ![image](https://user-images.githubusercontent.com/61408860/179551776-fe4cd3a1-8aba-45cf-bf1d-ecb7d06bbb29.png)

Reply:
@danmainah Hey there, I'm having this issue on my end while viewing on any viewport width and height. I'm not sure what would cause this on your end.